hgt/getHGTElevation.js

```diff
@@ -0,0 +1,47 @@
+// This file is largely derived from https://github.com/perliedman/node-hgt/blob/master/src/hgt.js
+module.exports = function getHGTElevation(hgt, coord) {
+    const size = hgt.size - 1;
+    const row = (coord[1] - hgt.swLngLat[1]) * size;
+    const col = (coord[0] - hgt.swLngLat[0]) * size;
+
+    if (row < 0 || col < 0 || row > size || col > size) {
+        throw new Error(
+            'Latitude/longitude is outside tile bounds (row=' +
+                row +
+                ', col=' +
+                col +
+                '; size=' +
+                size
+        );
+    }
+
+    // This could be changed to nearestNeighbor or configured with options,
+    // for example return hgt.options.interpolation.call(this, row, col);
+    return bilinear(hgt, row, col);
```

Comment:
I think either works here, though bilinear makes a lot of sense if we're using SRTM, which of course is quite coarse relatively.